### ISSUE TITLE
Accessibility [Inspect]: Cannot find the RichEdit control of the "Text" property in UIA structure of a PropertyGrid control

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#2704]
* .NET Core Version: 

.NET Core 5.0.0-alpha.1.19564.1 (also reproduces in earlier versions of .NET Core WinForms.

* Have you experienced this same bug with .NET Framework?: 
Yes

**Problem description:**

The issue is that RichEdit control is not shown in the UIA hierarchy of PropertyGrid control children.

![image](https://user-images.githubusercontent.com/23213980/72268000-0e09a800-3632-11ea-8570-a79c012f257f.png)

**Expected behavior:**
RichEdit control (when shown) should appear in the PropertyGrid control's children hierarchy.

**Minimal repro:**
* Create WinForms app with PropertyGrid and Button control.
* Associate the button with PropertyGrid as a selected object.
* Run the app, type something in the RichEdit control control (this is a button's text input control) and inpsect the button with the Inspect tool. Observe that the input is not in the UIA hierarchy.

**Comment by weltkante** [comment]
Does this work for other popup/dropdown editors in the property grid? (Control editors, including custom editors, have the option to opt-in into either a dropdown or a popup.) If the problem isn't just isolated to RichTextBox it might be worth looking for a general solution so user defined editors can also profit from the solution.

If the solution to this accessibility issue requires certain code in the editor itself it might be worth documenting it, so programmers of user-defined editors can also solve this problem for their own editor implementations.

**Comment by merriemcgaw** [comment]
That's a great point @weltkante .I _think_ this works as expected for the other editors. @M-Lipin can you confirm? If there's something specific we did to get those to work that we missed with this one we can certainly add that note to the docs. I've got an accessibility samples and documentation project simmering on a back burner that may be coming up one of these days :)

**Comment by M-Lipin** [comment]
Thank you for the points @merriemcgaw and @weltkante ! The solution works for all drop-down editors. But popup inner accessibility and accessible hierarchy is the responsibility of editor's developer.

**Comment by Marie-Shi** [comment]
Verified the issue with latest .NET 5.0.100-rc2.20459.18 from RC2 branch, it is **fixed**. Now RichEdit control (when shown) appears in the PropertyGrid control's children hierarchy.
![Untitled](https://user-images.githubusercontent.com/45864017/92705615-6ee6ad00-f386-11ea-98c2-ac7b5888a74f.png)

**Comment by Marie-Shi** [comment]
Verified the issue with latest .NET 6.0.100-preview.1.21080.4 from Release/6.0.1xx-preview1 branch, it is fixed. Now RichEdit control (when shown) appears in the PropertyGrid control's children hierarchy.
